.gitignore

```diff
@@ -10,3 +10,5 @@ a_run_front_end
 a_run_middle_layer
 nohup*
 secret*
+front_end/Containerfile
```

Comment:
We do want the front end container file to be in the GitHub repository. Maybe you intended to ignore something different?

Reply:
Please ignore the entire ".idea" directory.